openmetadata-ui/src/main/resources/ui/src/components/TasksDAGView/TasksDAGView.tsx

```diff
@@ -36,12 +36,16 @@ const TasksDAGView = ({ tasks, selectedExec }: Props) => {
   const [edgesData, setEdgesData, onEdgesChange] = useEdgesState([]);
 
   const getNodeType = useCallback(
-    (index: number) => {
-      return index === 0
-        ? 'input'
-        : index === tasks.length - 1
-        ? 'output'
-        : 'default';
+    (task: Task) => {
+      const isDownStreamTask = tasks.some((taskData) =>
+        taskData.downstreamTasks?.includes(task.name)
+      );
+
+      if (isDownStreamTask) {
+        return 'output';
+      } else {
+        return 'default';
+      }
```

Comment:
We have an enum for input, output and default can we use that here! let me know if you need help! you can also refer lineage code!

Reply:
Sure, @ShaileshParmar11 Thanks.